### ISSUE TITLE
GO: data load now takes 3 hours instead of 10 minutes

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#1774]
@rachellyne The go.obo file now takes 3 hours to load instead of 10 minutes. If I load the go-basic.obo file, it still is 10 minutes. 

What has changed? 

http://geneontology.org/page/download-ontology#Currently_available

```
go-basic.obo

This is the basic version of the GO filtered such that the graph is guaranteed to be acyclic, and 
annotations can be propagated up the graph. The relations included are is_a, part_of, regulates, 
negatively_regulates and positively_regulates. This version excludes relationships that cross the 3 
main GO hierarchies. This is the version that should be used with most GO-based annotation tools.

go.obo and go.owl

These files contain the core GO ontology in two formats, OBO and OWL-RDF/XML. This view includes 
relationships not in the filtered version of GO including has_part and occurs_in. The set of 
relationships used is likely to extend over time. Many of these relationships may not be safe for 
propagating annotations across, so this version should not be used with legacy GO tools. This version 
currently excludes relationships to external ontologies.

```

I guess they have added new relationship types? It's likely that InterMine is not processing these types correctly, so we should only load go-basic.obo.

See also: https://lists.intermine.org/pipermail/dev/2018-March/004019.html

TODO

* ~update docs~
* ~update download script~ - https://github.com/intermine/intermine-scripts/commit/bcdf674d4561865a1e65093ed3f39d863aa71377

**Comment by sammyjava** [comment]
Good to know! I've been loading GO for hours but then dumping it to a DB dump file for quicker loading. I'll use go-basic.obo from now on.

**Comment by julie-sullivan** [comment]
Fixed download script bcdf674d4561865a1e65093ed3f39d863aa71377

**Comment by julie-sullivan** [comment]
`Gene --> GO terms + Parents of those terms` returns different values based on which GO file you parse.

For the older GO file, when running the above template, there are 98 GO parents. In `go-basic`, there are only 70. 

Here is the default GO term for that template, `acetylcholine-gated cation-selective channel activity`: https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022848

Our data seems to match but better to know why there is a difference. 

@rachellyne Any ideas on this? Should we load larger file instead?
